File: server/src/main/kotlin/com/va034600/Controller.java
package com.va034600;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class Controller {
    @GetMapping("/pub/test")
    public String pub() {
        return "pub";
    }

    @GetMapping("/auth/test")
    public String auth() {
        return "auth";
    }
}

File: server/src/main/kotlin/com/va034600/Controller.kt
package com.va034600

import org.springframework.web.bind.annotation .GetMapping
import org.springframework.web.bind.annotation .RestController

@RestController
class Controller {
    @GetMapping("/pub/test")
    fun pub(): String {
        return "pub"
    }

    @GetMapping("/auth/test")
    fun auth(): String {
        return "auth"
    }
}